src/components/views/voip/PictureInPictureDragger.tsx

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright 2017, 2018 New Vector Ltd
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import React, { createRef } from 'react';
+import UIStore from '../../../stores/UIStore';
+import { IApp } from '../../../stores/WidgetStore';
+import { lerp } from '../../../utils/AnimationUtils';
+import { MarkedExecution } from '../../../utils/MarkedExecution';
+import { replaceableComponent } from '../../../utils/replaceableComponent';
+import { CallViewHeader } from './CallView/CallViewHeader';
```

Comment:
I believe we should rename that component. The `PictureInPictureDragger` appears to be quite generic and has nothing to do with `CallView`

Reply:
Unfortunately it's very tied to the calls for now, as the header is not generic so far (takes `App` or a `Call[]`). Need to figure it out, but I don't know enough about the codebase to properly separate it yet.